exporter_test.go

```diff
@@ -1114,6 +1114,37 @@ version = "4.5.6"
 				h.AssertNil(t, err)
 				h.AssertContains(t, fakeAppImage.SavedNames(), append(opts.AdditionalNames, fakeAppImage.Name())...)
 			})
+
+			when("there is build metadata", func() {
```

Comment:
"build metadata" seems like an overloaded term, perhaps "build.toml" would be better

Reply:
agree. unlike other files with configurable names, `build.toml` is always called `build.toml` so we might as well stick with that instead of wasting precious terms here that we might want to (or already) use for other concepts.